packages/yew-macro/src/props/element.rs

```diff
@@ -64,154 +64,148 @@ impl Parse for ElementProps {
-            // where `Value` = Boolean attribute
-            // Note: `checked` is uniquely handled in the html! macro.
-            "allowfullscreen",
-            "async",
-            "autofocus",
-            "autoplay",
-            "controls",
-            "default",
-            "defer",
-            "disabled",
-            "formnovalidate",
-            "hidden",
-            "ismap",
-            "itemscope",
-            "loop",
-            "multiple",
-            "muted",
-            "nomodule",
-            "novalidate",
-            "open",
-            "playsinline",
-            "readonly",
-            "required",
-            "reversed",
-            "selected",
-            "truespeed",
-        ]
-        .into_iter()
-        .collect()
-    };
-}
-
-lazy_static! {
-    static ref LISTENER_SET: HashSet<&'static str> = {
-        vec![
-            // Living Standard
-            // From: https://html.spec.whatwg.org/multipage/webappapis.html#globaleventhandlers
-            "onabort",
-            "onauxclick",
-            "onblur",
-            "oncancel",
-            "oncanplay",
-            "oncanplaythrough",
-            "onchange",
-            "onclick",
-            "onclose",
-            "oncontextmenu",
-            "oncuechange",
-            "ondblclick",
-            "ondrag",
-            "ondragend",
-            "ondragenter",
-            "ondragexit",
-            "ondragleave",
-            "ondragover",
-            "ondragstart",
-            "ondrop",
-            "ondurationchange",
-            "onemptied",
-            "onended",
-            "onerror",
-            "onfocus",
-            // onfocusin + onfocusout not in standard but added due to browser support
-            // see issue 1896: https://github.com/yewstack/yew/issues/1896
-            "onfocusin",
-            "onfocusout",
-            "onformdata",
-            "oninput",
-            "oninvalid",
-            "onkeydown",
-            "onkeypress",
-            "onkeyup",
-            "onload",
-            "onloadeddata",
-            "onloadedmetadata",
-            "onloadstart",
-            "onmousedown",
-            "onmouseenter",
-            "onmouseleave",
-            "onmousemove",
-            "onmouseout",
-            "onmouseover",
-            "onmouseup",
-            "onpause",
-            "onplay",
-            "onplaying",
-            "onprogress",
-            "onratechange",
-            "onreset",
-            "onresize",
-            "onscroll",
-            "onsecuritypolicyviolation",
-            "onseeked",
-            "onseeking",
-            "onselect",
-            "onslotchange",
-            "onstalled",
-            "onsubmit",
-            "onsuspend",
-            "ontimeupdate",
-            "ontoggle",
-            "onvolumechange",
-            "onwaiting",
-            "onwheel",
+static BOOLEAN_SET: Lazy<HashSet<&'static str>> = Lazy::new(|| {
```

Comment:
We don't need lazy for this. HashSet can be constructed for pre-defined elements in a const context with [HashSet::from](https://doc.rust-lang.org/std/collections/struct.HashSet.html#impl-From%3C%5BT%3B%20N%5D%3E)

Reply:
`HashSet::from` is not `const fn`.

As far as I know, `HashSet` needs a `RandomState` which will not be able to be initialised until a after a thread is available. So still needs `Lazy::new`.

Reply:
I see, you're right.